sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/auth/DefaultAuthorizationInterceptor.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
```

Comment:
Should change to `1999-2021` here?

Reply:
Both are OK, since there are remain other files which marked as `1999-2018`, we may uniform them in the future.